src/i_oalmusic.c

```diff
@@ -429,6 +465,28 @@ static void *I_OAL_RegisterSong(void *data, int len)
                                             &player.freq, &player.frame_size))
         {
             active_module = all_modules[i];
+
+            int channels = AL_FORMAT_STEREO16;
+            switch (player.format)
+            {
+                case AL_FORMAT_MONO16:
+                case AL_FORMAT_MONO_FLOAT32:
+                    channels = 1;
+                    break;
+                case AL_FORMAT_STEREO16:
+                case AL_FORMAT_STEREO_FLOAT32:
+                    channels = 2;
+                    break;
+            }
+
+            ebur128 = ebur128_init(channels, player.freq, EBUR128_MODE_M);
+
+            if (channels == 2)
```

Comment:
What if channels > 2?

Reply:
We don't allow it for music. I have never seen such files.